packages/amplication-server/migration-scripts/install-auth-core-plugin.ts

```diff
@@ -82,42 +84,108 @@ async function main() {
     await migrateChunk(chunk);
   }
 
+  function isPluginExist(
+    plugins: PluginInstallation[],
+    authProvider: string
+  ): boolean {
+    const plugin = plugins.find(
+      (plugin) => plugin.pluginId.trim() == authProvider
+    );
+
+    if (plugin) return true;
+    return false;
```

Comment:
we actually can use the `.some()` method that returns boolean: true if it found the item and false if it didn't find
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

```suggestion
    return plugins.some(
      (plugin) => plugin.pluginId.trim() == authProvider
    );
```

Reply:
But it's just a suggestion...we can leave it like that